thunder/executors/torchex.py

```diff
@@ -851,11 +855,33 @@ def _elementwise_unary_with_inplace_checker(a: TensorProxy, /, inplace: bool = F
 _register_elementwise_unary_implementation(ltorch.elu, elu, checker=_always_executable)
 _register_elementwise_unary_implementation(ltorch.celu, celu, checker=_always_executable)
 _register_elementwise_unary_implementation(ltorch.gelu, gelu, checker=_always_executable)
+_register_elementwise_unary_implementation(ltorch.hardshrink, hardshrink, checker=_always_executable)
+_register_elementwise_unary_implementation(ltorch.hardswish, hardswish, checker=_elementwise_unary_with_inplace_checker)
 _register_elementwise_unary_implementation(ltorch.leaky_relu, leaky_relu, checker=_always_executable)
+_register_elementwise_unary_implementation(
+    ltorch.log_sigmoid_backward, log_sigmoid_backward, checker=_always_executable
+)
+# _register_elementwise_unary_implementation(ltorch.logsigmoid, logsigmoid)
+
+
+def log_sigmoid_grad_transform(a):
```

Comment:
@mruberry  do you mean something like this?

Reply:
Yes - but this would need to include both the CPU and CUDA paths because the PyTorch executor can run both CPU and CUDA operations (if it's set as the only executor, for example, it will run CUDA operations)

Reply:
(I would leave the follow-up for a separate PR)